### PR TITLE
Added a new MathsPuzzle class

### PR DESCRIPTION
**Pull request by chasinglogic**

### CODE REVIEW

File: entities/maths_puzzle.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from entities.puzzle import Puzzle
+from random import randint
+
+
+class MathsPuzzle(Puzzle):
+    def __init__(self):
+        rand1 = randint(1, 100)
+        rand2 = randint(1, 100)
+
+        self.question = f"what is {rand1} + {rand2}?"
```
Comment (chasinglogic) [inline entities/maths_puzzle.py:11]: Could we make the operation random as well?

**Review by 2Steaks — APPROVED** [review]

File: entities/maths_puzzle.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        self.answer = str(rand1 + rand2)
+        self.rand3 = str(randint(1, 200))
+        self.rand4 = str(randint(1, 200))
```
Comment (davidttest) [inline entities/maths_puzzle.py:14]: Could we have the same answer offered more than once (if we're very unlucky)?

**Review by davidttest — APPROVED** [review]
Minor comment

File: entities/maths_puzzle.py (hunk 2 above)
Comment (chasinglogic) [reply entities/maths_puzzle.py:14]: @davidttest can you open a bug please.

**Pull request merged**